.github/workflows/test.yml

```diff
@@ -53,15 +53,23 @@ jobs:
       - name: Install Dependencies and lightly
         run: pip install -e '.[all]'
       - name: Clean Previous Coverage Data
+        if: matrix.python == '3.10'
```

Comment:
I am not sure about this. Why not alway remove it?

Reply:
Removed it